Sources/SyrupCore/Generator/Generator.swift

```diff
@@ -142,26 +147,26 @@ public final class Generator {
 		}
 	}
 
-	public static func parseOperations(graphQLString: String) throws -> (queries: [String: String], mutations: [String: String], fragments: [String: String]) {
+	public static func parseOperations(graphQLString: String) throws -> (queries: [String: String], mutations: [String: String], fragments: [String: String], subscriptions: [String: String]) {
```

Comment:
Nitpick, but wouldn't `subscriptions` be a more logical 3rd return value (`queries, mutations, subscriptions, fragments`).

Reply:
I shuffled these around and it was generating zero diffs but the tests were still failing. zero useful output 🤷

Reply:
actually they just fail no matter what now for me now

Reply:
ok mysteriously fixed now and in the correct order